### PR TITLE
Add CMake command to run shellcheck.

### PR DESCRIPTION
**Pull request by pattivacek**
Note that this isn't quite the same as what CI is doing, since that has some extra magic to find shell scripts that don't end in `.sh`. I don't want to reimplement that here. However, I renamed the files I know about that didn't have that extension so that they have it, just to make it easier to find them.

Also, bump tuf-test-vectors and switch to uptane namespace.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -289,6 +289,12 @@ function(aktualizr_source_file_checks)
     endif()
 endfunction()
 
+add_custom_target(shellcheck
+    COMMAND find docs/ scripts/ src/ tests/ -name "*.sh" | xargs shellcheck
+    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+    VERBATIM)
+add_dependencies(qa shellcheck)
```
Comment (cajun-rat) [inline CMakeLists.txt:296]: What about:

```
find_program(SHELLCHECK NAMES shellcheck)
if(SHELLCHECK)
    add_custom_target(shellcheck
        COMMAND find docs/ scripts/ src/ tests/ -name "*.sh" | xargs ${SHELLCHECK}
        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
        VERBATIM)

    add_dependencies(qa shellcheck)
else()
    message(WARNING "shellcheck not found, skipping")
endif()
```

Comment (pattivacek) [reply CMakeLists.txt]: Makes sense. Done.

**Comment by cajun-rat** [comment]
This is weird. I was going to rebased this on top of the buildfix so that we can get it in, but when I ran it locally I get a bunch of errors in `makedeployed.sh` and `makephysical.sh` like:

```
In tests/ostreesysroot/makephysical.sh line 51:
cd ${CURDIR}
   ^-------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
cd "${CURDIR}"
```
The shellcheck CI rule passes, which is weird. It appears to be checking those files (it says it is...). I'm running the version in Debian Stable which is 0.7.1, vs 0.8.0 in CI which might be the issue. I'm happy to put those fixes in, but it weird that it passes today?

**Comment by pattivacek** [comment]
> This is weird. I was going to rebased this on top of the buildfix so that we can get it in, but when I ran it locally I get a bunch of errors in `makedeployed.sh` and `makephysical.sh` like:
> 
> ```
> In tests/ostreesysroot/makephysical.sh line 51:
> cd ${CURDIR}
>    ^-------^ SC2086: Double quote to prevent globbing and word splitting.
> 
> Did you mean: 
> cd "${CURDIR}"
> ```
> 
> The shellcheck CI rule passes, which is weird. It appears to be checking those files (it says it is...). I'm running the version in Debian Stable which is 0.7.1, vs 0.8.0 in CI which might be the issue. I'm happy to put those fixes in, but it weird that it passes today?

That is weird. This was actually blocked because of the tuf-test-vectors namespace change that was failing to fetch for reasons I couldn't figure out and still haven't had time to investigate. Maybe I should separate the two pieces.

**Comment by pattivacek** [comment]
I removed the tuf-test-vectors fix and put it in https://github.com/uptane/aktualizr/pull/73 so that this can proceed.

**Comment by pattivacek** [comment]
grrrr what happened with codecov again? It was working just a few weeks ago, but now it's not posting anymore again. :(

**Comment by pattivacek** [comment]
Can I get a review on this?

**Review by mike-sul — APPROVED** [review]

**Pull request merged**